source/MapDetailPanel.cpp

```diff
@@ -589,6 +614,31 @@ void MapDetailPanel::DrawInfo()
 		uiPoint.Y() += 20.;
 	}
 
+	autoBuyY = uiPoint.Y();
+
+	// "Buy best" button goes after trade prices
+	if(player.CanTrade())
+	{
+		int canBuyBest = player.HasBestTrade(selectedSystem);
+		if(canBuyBest || player.Cargo().CommoditiesSize())
+		{
+			const Sprite *buyAllSprite = SpriteSet::Get("ui/planet dialog button"); //160x60
+			const auto alignCenter = Layout(160, Alignment::CENTER);
+			SpriteShader::Draw(buyAllSprite, uiPoint + Point(70, 35));
+			if(canBuyBest == 0)
+			{
+				font.Draw({"Sell Commodi_ties", alignCenter}, uiPoint + Point(-10, 28), medium);
+			}
```

Comment:
```suggestion
				font.Draw({"Sell Commodi_ties", alignCenter}, uiPoint + Point(-10, 28), medium);
```
No need to use braces if it's just one line

Also maybe add a comment explaining that the `_` is there to show that t is the key to press to do it?

Reply:
I had thought the underscore was just a part of the text system - e.g. the ShopPanel `string BUY = IsAlreadyOwned() ? (playerShip ? "_Install" : "_Cargo") : "_Buy";`

Reply:
Yeah it is, you don't have to add it no worries
It's there to show what key you should press to do a given thing; in those cases I, C, B